src/hooks/queries/useHomeGalleryAssets.js

```diff
@@ -91,21 +91,33 @@ const useHomeGalleryAssets = () => {
     if (img.src.includes('1.jpg')) {
       img.width = LARGE_PICTURE.WIDTH;
       img.height = LARGE_PICTURE.HEIGHT;
+      img.alt = 'workspace';
```

Comment:
can you write relevant alts and titles?

Reply:
1) Pleasant working atmosphere
2) Cozy workplace